math/mathcore/src/TMathVectorized.cxx

```diff
@@ -331,7 +331,116 @@ Double_v TMath::LaplaceDistI(Double_v x, Double_t alpha, Double_t beta)
 }
 
 // Double_v TMath::LogNormal(Double_v x, Double_t sigma, Double_t theta=0, Double_t m=1);
-// Double_v TMath::NormQuantile(Double_v p);
+
+Double_v TMath::NormQuantile(Double_v p)
+{
+   // Ensure all input probabilities are valid
+   vecCore::Mask_v<Double_v> invalid_values = (p <= 0) || (p >= 1);
```

Comment:
I thought so, but I just translated what I saw in the original implementation of NormQuantile: https://github.com/root-project/root/blob/master/math/mathcore/src/TMath.cxx#L2404 Anyway, I removed this vectorized implementation in the last commit because it had some precision error: linear and vectorized versions compute the same results but some of them have a difference up to 1e-4. As this function does not seem very useful for the fitting methods we want to vectorize, we decided to postpone its implementation.